packages/react/rollup.config.js

```diff
@@ -3,7 +3,7 @@ import fs from 'fs';
 
 import includePaths from 'rollup-plugin-includepaths';
 import resolve from '@rollup/plugin-node-resolve';
-import ts from '@wessberg/rollup-plugin-ts';
+import ts from 'rollup-plugin-ts';
```

Comment:
I guess this library delivers but I was thinking can we use @rollup/plugin-typescript here?
Is it missing something?

Reply:
I was wondering that too that why was this chosen instead of the rollup plugin. I didn't dare to try and change it 🙈 I can test it out and search why people would use this.